docs/UserGuide.md

```diff
@@ -146,12 +147,103 @@ Invalid usages
 The commands in this section are used to view and retrieve records on the system, such as students, tutorials, and enrollment status.
 
 - [Listing all students](#41-listing-all-students)  
-4.2 Search
+- [Searching students](#42-searching-students)
 
 #### **4.1 Listing all students**
 
 Command: `list`
 
+<br>
+
+#### **4.2 Searching students**
+
+Search for students that match the condition specified by the given prefix.
+
+<div markdown="span" class="alert alert-success">:bulb:
+**Tip:**
+
+This command is best used with reference to its usages in the [Prefix Table](#11-prefix-table-)
+</div>
+
+
+Command:  `find`
+
+Usage: `find [n/NAME] [e/EMAIL] [p/PHONE] [a/ADDRESS] [pay/PAYMENT] [a/ATTENDANCE] [t/TAG] [tut/TUTORIAL_NAME]`
+
+<div markdown="span" class="alert alert-primary">:pushpin: **Note:**
+
+1. Fields wrapped in square brackets `[]` are optional, but at least one of the optional fields must be provided.
```

Comment:
Should we include so much content inside note? 
Would it be better to put them in fields instead?